.settings/org.eclipse.wst.common.project.facet.core.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ditto: this file needs to be rolled back. In future don't add files unless you know they're supposed to be different.

Reply:
I have a `~/.gitignore_global` file that contains files I in general want to ignore. They can always be added if needed. The most important ones are the . files.
```
patchLabels.py
.externalToolBuilders/
/rdkit/Chem/inchi.py
/.project
/.cproject
/.classpath
/.texlipse
/.pydevproject
*.so
*.swp
*.pyc
/.pydevproject/
/lib/*
.DS_Store
/.settings/*
/Debug/
/src/log4j2.xml
/.scannerwork
/sonar-project.properties
/jacoco.exec
/baeconsole.pg
```

Reply:
I thought the .gitignore mechanism had no effect on files that are already staged? Things like .classpath actually need to be in the project, but they need to not be modified gratuitously if you had to tweak it to match your own unique configuration...

Reply:
That is the case. It's more relevant for the .settings directory and some other stuff that I in general want to exclude.